geopoint/operations/default.json

```diff
@@ -453,11 +453,57 @@
         }
       }
     },
+    {
+      "name": "distanceFilterCount200x300-alt",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+              "geo_shape": {
+                "location": {
+                    "shape": { "type": "circle", "radius": "300km", "coordinates": [7, 55] },
+                    "relation": "intersects"
+                  }
+                }
+              },
+              {
+                "bool": {
+                  "must_not": [
```

Comment:
Instead of a must not, could we use a disjoint spatial relationship?

Reply:
The disjoint version was the original version. This is an alternative query that deliberately does things differently, to see if that makes a difference to the performance. It does not seem to make any difference.